tests/parser/ast_utils/test_ast_dict.py

```diff
@@ -0,0 +1,81 @@
+from vyper import (
+    compiler,
+)
+from vyper.ast_utils import (
+    ast_to_dict,
+    dict_to_ast,
+    parse_to_ast,
+)
+
+
+def get_node_ids(ast_struct, ids=None):
+    if ids is None:
+        ids = []
+
+    for k, v in ast_struct.items():
+        print(k, v, ids)
```

Comment:
Much better to use `logging` statements in tests so that the runner can control the output verbosity.